Lib9c/Action/Garages/DeliverToOthersGarages.cs

```diff
@@ -0,0 +1,265 @@
+#nullable enable
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Security.Cryptography;
+using Bencodex.Types;
+using Lib9c.Abstractions;
+using Libplanet;
+using Libplanet.Action;
+using Libplanet.Assets;
+using Libplanet.State;
+using Nekoyume.Exceptions;
+using Nekoyume.Model.Garages;
+using Nekoyume.Model.State;
+
+namespace Nekoyume.Action.Garages
+{
+    [ActionType("deliver_to_others_garages")]
```

Comment:
Good point. I thought about how well the name of the action should indicate the function.
Agents can actually have many garages. This is because individual garages are used depending on the type of currency or item.
So the `DeliverToOthersGarages` action can change the state of the other agent's various garages, which is reflected in the action's name.

Shouldn't the name of an action reflect the specifics of the state it changes? Conversely, if you omit the `s`, it is likely to create a misunderstanding that only one garage state is changed.

I think like this. If the garage is written as a function module rather than a state, and the unit of state stored by currency or item is a box rather than a garage, it would be more natural to subtract `s` from the action name. What is this direction like?